frontend/src/components/housing/submission/SubmissionIntakeForm.vue

```diff
@@ -1046,6 +1020,51 @@ onBeforeMount(async () => {
                 </div>
               </template>
             </Card>
+            <Card>
+              <template #title>
+                <span class="section-header">Help us learn more about your housing project</span>
+                <Divider type="solid" />
+              </template>
+              <template #content>
+                <div class="col-12 mt-0 pt-0 mb-0 pb-0">
+                  <div class="flex align-items-center">
+                    <label>Provide additional information</label>
+                    <div
+                      v-tooltip.right="
+                        `Provide us with additional information -
+                         short description about the project, project website link, or upload a document.`
+                      "
+                      class="pl-2 mb-2"
+                    >
+                      <font-awesome-icon icon="fa-solid fa-circle-question" />
+                    </div>
+                  </div>
+                </div>
+                <!-- eslint-disable max-len -->
```

Comment:
Style: We should not be sprinkling in directives like this. This will disable checking for max-len for the rest of the file. This should only be done on a line-by-line basis.